client/src/components/Checkout/checkout.js

```diff
@@ -44,53 +45,57 @@ function goBack() {
   window.location.href="/browse";
```

Comment:
Ideally goBack() would put them back in the page they were before visiting /checkout?